frontend/src/components/VersionSelector/VersionSelector.tsx

```diff
@@ -89,12 +108,17 @@ export const VersionSelector: FC<VersionSelectorProps> = ({
       return (
         v.uuid === currentVersion?.uuid ||
         v.uuid === conceptVersion?.uuid ||
-        v.uuid === selectedVersionId
+        v.uuid === currentVersionUUID
```

Comment:
Maybe the "selected" in the name made it clearer that this is different from the "current version" aka the "latest active version"